odl/set/pspace.py

```diff
@@ -256,15 +258,16 @@ def element(self, inp=None):
         if inp is None:
             inp = [space.element() for space in self.spaces]
 
-        # TODO: how does this differ from "if inp in self"?
```

Comment:
In response to whoever wrote this, this simply verifies that the parts are correct, perhaps we only got a tuple, or we got an element in some other product space.